SettingsList.py

```diff
@@ -2744,6 +2744,34 @@ def __init__(self, name, gui_text, min, max, default, step=1,
         choices        = HintDistList(),
         gui_tooltip    = HintDistTips(),
         shared         = True,
+        disable        = {
+            'balanced'  : {'settings' : ['bingosyncURL']},
+            'strong'  : {'settings' : ['bingosyncURL']},
+            'tournament'  : {'settings' : ['bingosyncURL']},
+            'useless'  : {'settings' : ['bingosyncURL']},
+            'very_strong'  : {'settings' : ['bingosyncURL']}
+        },
+    ),
+    Setting_Info(
+        name           ="bingosyncURL",
+        type           =str,
+        choices        ={},
+        gui_type       ="Textinput",
+        gui_text       ="Bingosync URL",
+        shared         = True,
+        gui_tooltip    = '''\
+            Enter a URL to a Bingosync bingo board in order to have hints specific to items needed to beat the board. Goals which involve finding an item are not directly hinted.
```

Comment:
Replaced by "Goals which are completed simply by finding a specific item are not hinted (e.g. "Boomerang"). "

There are some bingo goals that are simply to collect a certain item (goals like "Megaton Hammer" or "At least 2 elemental arrows"). Since it would entirely defeat the point of the goal to have a hint telling you where those items are, those types of goals do not add any extra hints.